ui_testing/testcases/basic_tests/test06_orders.py

```diff
@@ -2296,3 +2296,26 @@ def test065_download_suborder_sheet_for_single_order(self):
                                   f"{str(fixed_sheet_row_data)} : {str(formatted_orders[index])}")
             for item in formatted_orders[index]:
                 self.assertIn(item, fixed_sheet_row_data)
+                
+    @skip("https://modeso.atlassian.net/browse/LIMSA-236")
+    def test067_create_existing_order_change_contact(self):
+        """
+       LIMS-4293: Orders: Table: Existing order: Create existing order
+       then change the contact for this existing one, all old records with the same order number will update its contact.
+        """
+        order_no = self.order_page.create_existing_order_with_auto_fill(no='')
+        self.order_page.sleep_tiny()
+        new_contact = self.order_page.set_contact(remove_old=True)[0]
```

Comment:
you can get all suborders data and save analysis number of them.
Then  in analysis page , you can filter by analysis number and make sure Thar contact = new contact. So, test case won't be skipped any more

Reply:
done

Reply:
thanks :)